conans/paths/simple_paths.py

```diff
@@ -40,25 +39,11 @@ def __init__(self, store_folder):
     def store(self):
         return self._store_folder
 
-    def _build_path_to_base_folder(self, ref):
-        return os.path.normpath(os.path.join(self.store, ref.dir_repr()))
-
-    def _build_path_to_linked_folder_sentinel(self, ref):
-        base_folder = self._build_path_to_base_folder(ref)
-        linked_package_file = os.path.join(base_folder, LINKED_PACKAGE_SENTINEL)
-        return linked_package_file
-
     def package_layout(self, ref, short_paths=False):
```

Comment:
This function is overriden in the `ClientCache` class, I think that this implementation is no longer used.

Reply:
It is being used by the server, that is not using ClientCache, but SimplePaths class